tools/src/wyvern/tools/errors/ErrorMessage.java

```diff
@@ -109,16 +109,20 @@
     NO_ABSTRACT_TYPES_IN_OBJECTS("Abstract types may not be declared in objects or modules, only in type definitions", 0),
     METHODS_MUST_BE_INVOKED("Cannot access a method as if it were a field; use ()", 0),
     TSL_ERROR("Error in type-specific language: %ARG", 1),
-    CANNOT_APPLY_TYPE_PARAMETERS("Cannot apply type parameters: type %ARG is abstract", 1),
-    NO_TYPE_MEMBER("Cannot find enough type members to apply type parameters", 0),
+    CANNOT_APPLY_GENERIC_ARGUMENTS("Cannot apply generic arguments: type %ARG is abstract", 1),
```

Comment:
The message name is about generics, but the message itself is still only about types.

Reply:
The generic arguments could be effects, not just types. The error message is saying that %ARG as a type is itself abstract; i.e., that we can't even look inside it to see what abstract members (abstract types, abstract effects) we could apply the generic arguments to.